doc/tut1.rst

```diff
@@ -185,9 +185,27 @@ to a storage location:
   var x = "abc" # introduces a new variable `x` and assigns a value to it
   x = "xyz"     # assigns a new value to `x`
 
-``=`` is the *assignment operator*. The assignment operator cannot be
-overloaded, overwritten or forbidden, but this might change in a future version
-of Nim. You can declare multiple variables with a single assignment
+``=`` is the *assignment operator*. The assignment operator can be
+overloaded:
```

Comment:
This example should NOT make it into the tutorial and in fact how ``=`` works needs to be rethought.

Reply:
Ok, removed.